SwaggerGen/Swagger/Type/ObjectType.php

```diff
@@ -108,6 +109,9 @@ public function handleCommand($command, $data = null)
 	{
 		switch (strtolower($command)) {
 			// type name description...
+			case 'discriminator':
+				$this->discriminator = $data;
```

Comment:
Specs say

> the discriminator field MUST be a required field.

Please validate `$data` is a required field and throw an exception if not.

Reply:
Good point, will do. I'm leaning towards the automatically putting this into 'required' list though, as opposed to requiring the consumer code to do this separately.

Reply:
That's probably a better solution, given that SG generally tries to be lenient with the author unless leniency would limit expressiveness

Reply:
>  I'm leaning towards the automatically putting this into 'required' list 

nope, that's probably wouldn't work, as `discriminator typeProp` is not enough, explicit property with type would also be required.